docs/examples/creating-accounts.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Creating Accounts
```

Comment:
Would be good to mention Flow and the Go SDK in the titles if possible -- better for SEO and link previews.

You can use the `sidebar_title` to specify a shorter title that only shows up in the side nav.

```suggestion
title: Creating Flow Accounts from Golang
sidebar_title: Creating Accounts
```